src/content/LandingPage/_landing-page.scss

```diff
@@ -0,0 +1,49 @@
+@import './mixins.scss';
+@import './overrides.scss';
+
+.landing-page__illo {
+  max-width: 100%;
+}
+
+.landing-page__heading {
+  @include carbon--type-style('productive-heading-05');
+}
+
+.landing-page__r2 {
+  margin-top: rem(-40px);
+}
+
+.landing-page__tab-content {
+  padding-top: $layout-05;
+  padding-bottom: $layout-05;
+}
+
+.landing-page__banner {
+  padding-top: $spacing-05;
+  padding-bottom: $spacing-07 * 4;
```

Comment:
You're missing the background mixin for the banner row.  See https://www.carbondesignsystem.com/tutorial/react-step-2#banner